src/foam/u2/view/UnstyledTableView.js

```diff
@@ -673,6 +663,23 @@ foam.CLASS({
 
               return tbodyElement;
             });
+
+            if ( this.memento && this.memento.paramsObj.c ) {
+              var columns = this.memento.paramsObj.c.split(',');
+              for ( var c of columns ) {
+                if ( this.shouldColumnBeSorted(c) && ! c.includes('.')) {
+                  var prop = view.props.find(p => p.fullPropertyName === c.substr(0, c.length - 1) );
+                  if ( prop ) {
+                    if ( c[c.length - 1] === this.DESCENDING_ORDER_CHAR )
+                      this.order = this.DESC(prop.property);
+                    else
+                      this.order = prop.property;
+                    dao = dao.orderBy(this.order);
+                  }
```

Comment:
Should you break out of the loop at this point?

Reply:
I'm not sure why would we. 
we are going through properties that have sorting order char for which we set order and do sorting and then we go to the next one